extensions/blocks/donations/stripe-nudge.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+// eslint-disable-next-line wpcalypso/import-docblock
+import { Button, Dashicon } from '@wordpress/components';
+import { Warning } from '@wordpress/block-editor';
+import { compose } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { withDispatch } from '@wordpress/data';
+
+const StripeNudge = ( { autosaveAndRedirect, stripeConnectUrl } ) => (
```

Comment:
Could we rely on the existing component for this?
https://github.com/Automattic/jetpack/blob/master/extensions/shared/components/stripe-nudge/index.jsx